lib/stdlib/src/edlin_expand.erl

```diff
@@ -49,9 +49,9 @@ expand_module_name(Prefix) ->
 expand_function_name(ModStr, FuncPrefix) ->
     case to_atom(ModStr) of
 	{ok, Mod} ->
+		L = Mod:module_info(),
```

Comment:
Perhaps the best fix then is to replace `erlang:module_loaded/1` by `code:ensure_loaded/1`?

Reply:
Hello,
currently trying to complete a non existing module fails like this : 
```
1> trrr:*** ERROR: Shell process terminated! ***
=ERROR REPORT==== 15-Jul-2019::14:33:36.818327 ===
Error in process <0.65.0> with exit value:
{undef,[{trrr,module_info,[],[]},
        {edlin_expand,expand_function_name,2,
                      [{file,"/usr/lib/erlang/lib/stdlib-3.8.1/src/edlin_expand.erl"},
                       {line,52}]},
        {group,get_line1,5,[{file,"group.erl"},{line,596}]},
        {group,get_chars_loop,9,[{file,"group.erl"},{line,471}]},
        {group,io_request,6,[{file,"group.erl"},{line,181}]},
        {group,server_loop,3,[{file,"group.erl"},{line,117}]}]}

Eshell V10.3.2  (abort with ^G)
```

Reply:
this restart a shell. Indeed not very clever. I will considere your option.